addons/rivet/devtools/dock/settings.gd

```diff
@@ -31,3 +34,22 @@ func _populate_namespace_data(data: Object) -> void:
 	
 	%AuthNamespaceSelector.namespaces = namespaces
 	%DeployNamespaceSelector.namespaces = namespaces
+
+func _on_manage_version_button_pressed() -> void:
+		%ManageVersions.disabled = true
+		var result := await RivetDevtools.get_plugin().cli.run_command([
+			"--api-endpoint",
```

Comment:
This could be populated by the `RIVET_API_ENDPOINT` env var in the `run_command` fn right?

Reply:
Added in https://github.com/rivet-gg/plugin-godot/pull/6

Reply:
Fixed in: https://github.com/rivet-gg/plugin-godot/pull/6